wp-parsely.php

```diff
@@ -15,8 +15,8 @@
 
 // The default version is the first entry in the SUPPORTED_VERSIONS list.
 const SUPPORTED_VERSIONS = [
-	'2.6',
 	'3.0',
+	'2.6',
```

Comment:
We should consider moving the default version change to its own PR. It would make a revert much cleaner and faster should we need it.

```suggestion
	'2.6',
	'3.0',
```